README.md

```diff
@@ -52,6 +52,13 @@ security activities for Node.js core but relies on the Node.js Security Working
 Group to recommend and help maintain policies and procedures for that
 management.
 
+### Node.js Bug Bounty Program
+
+The Node.js project engages in an official bug bounty program for security researches and responsible public disclosures.
+
+The program is managed through the HackerOne platform at [https://hackerone.com/nodejs](https://hackerone.com/nodejs) with further details.
```

Comment:
if the URL is the same as the text, how about just putting it in angle brackets like:

```
the HackerOne platform at <https://hackerone.com/nodejs>
```

Reply:
In other places we just use the URL so I'll change it to plain text